UnityProject/Assets/Scripts/Items/Others/Magical/ScrollOfTeleportation.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections;
+using UnityEngine;
+using Mirror;
+using Systems.Spells.Wizard;
+
+namespace Items.Scrolls.TeleportScroll
+{
+	public class ScrollOfTeleportation : Scroll
+	{
+		private HasNetworkTabItem netTab;
+		private TeleportSpell teleport;
+
+		protected override void Awake()
+		{
+			base.Awake();
+			netTab = GetComponent<HasNetworkTabItem>();
+			teleport = GetComponent<TeleportSpell>();
+		}
+
+		public override bool WillInteract(HandActivate interaction, NetworkSide side)
+		{
+			if (base.WillInteract(interaction, side) == false) return false;
+
+			// If charges remain, return false to allow the HasNetworkTabItem component to take over.
+			return !HasCharges;
+		}
+
+		public void TeleportTo(TeleportDestination destination)
+		{
+			var teleportingPlayer = netTab.LastInteractedPlayer();
+
+			if (!HasChargesRemaining(teleportingPlayer)) return;
+
+			if (teleport.IsBusy)
+			{
+				Chat.AddExamineMsgFromServer(teleportingPlayer, $"You are already teleporting!");
+				return;
+			}
+
+			Transform spawnTransform = PlayerSpawn.GetSpawnForJob((JobType)destination);
+			teleport.ServerTeleportWizard(teleportingPlayer, spawnTransform.position.CutToInt());
+			Chat.AddChatMsgToChat(teleportingPlayer.Player(), $"SCYAR NILA {destination.ToString().ToUpper()}!", ChatChannel.Local);
```

Comment:
Codacy found an issue: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6316546)